docs/mkDocs/docs/specifications.md

```diff
@@ -1157,7 +1157,7 @@ types**.
 
 ### Definition
 
-A **stratified type** is a data type refined with `Prop` from
+A **stratified type** is a data type indexed with `Ix` from
 `Language.Haskell.Liquid.ProofCombinators`, where all recursive
 occurrences of the type:
 - appear under `Prop`, and
```

Comment:
There are still some mentions of Prop in the documentation.
```
docs/mkDocs/docs/options.md:     VV : {VV : Bool | Prop VV <=> Set_mem ?b (listElts ?a)}
docs/mkDocs/docs/specifications.md:data Map k a <l :: k -> k -> Prop, r :: k -> k -> Prop>
docs/mkDocs/docs/specifications.md:- appear under `Prop`, and
```

It is looking like those should be removed, but only the last one is related to this PR.